noggin/controller/registration.py

```diff
@@ -40,26 +66,26 @@ def _handle_registration_validation_error(username, e):
 
 def handle_register_form(form):
     username = form.username.data
-    password = form.password.data
     now = datetime.datetime.utcnow().replace(microsecond=0)
 
-    # First, create the user.
+    # First, create the stage user.
     try:
-        ipa_admin.user_add(
+        user = ipa_admin.stageuser_add(
             username,
             form.firstname.data,
             form.lastname.data,
-            f'{form.firstname.data} {form.lastname.data}',  # TODO ???
-            user_password=password,
+            # f'{form.firstname.data} {form.lastname.data}',  # TODO ???
```

Comment:
DO we need this commented out still (and the 'todo ???') or can it just be deleted?

Reply:
Yeah I'll change it, I wanted to leave it there because there's an inconsistancy in the python_freeipa library: `user_add` requires the fullname, but `stageuser_add` does not. Anyway leaving a commented line is certainly not the way to remember this ;-)